### ISSUE TITLE
[Update] Add isButton param and remove accessibilityLabel parameters to NavigationBarIcon in NAVIGATION_BAR

### ISSUE THREAD

**Issue by DevPabloGarcia** [Telefonica/webview-bridge#233]
**Message type**: `NAVIGATION_BAR `

**Originator**: Native

**Min app version**: 25.12

## Description

We need to remove two new optional parameters to the NavigationBarIcon type in the updateNavigationBar bridge method to improve accessibility and interaction handling:

**isButton**: A boolean flag to indicate whether the icon should be treated as a button (for semantic and accessibility purposes) | defaultValue = true

**name**: the accessibility label to add to the element. We use empty value to indicate the element should not has accessibility

~~accessibilityLabel: A string to provide a custom accessibility label for screen readers, replacing the deprecated name field~~

## Payload definition

**Request Payload**  

```ts
export type NavigationBarIcon = Readonly<{
      ...
      name: string;
      isButton: true | false | null
      ...
  }
```

Examples:

```json
{
    "type": "NAVIGATION_BAR", 
    "id": "web-2", 
    "payload": {
        "title": "My Page",
        "rightActions": [
            {
                "id": "help-icon",
                "name": "",
                "isButton": false,
                ...
            },
            {
                "id": "cart-icon",
                "name": "Shopping cart with 3 items",
                "isButton": true,
                ...
            }
        ]
        ...
    }
}
```

**Response Payload**  
The current ones.

## Possible error cases
The current ones.
